nicoleWojewoda/assignments/slidingwindow/lc53/lc53.js

```diff
@@ -1,5 +1,22 @@
 function max_subarray_size_k(k, arr) {
   // TODO: Write your code here
+  if (k > arr.length || k == 0) return 0;
+
+  let maxSum = 0,
+    currentSum = 0;
+
+  for (let i = 0; i <= arr.length - k; i++) {
+    currentSum = 0;
+    for (let j = i; j < k + i; j++) {
+      currentSum += arr[j];
+    }
+
+    if (currentSum > maxSum) {
+      maxSum = currentSum;
+    }
+  }
+
+  return maxSum;
 }
```

Comment:
Good work on the bruteforce approach! This is an approach that you have to becareful with because most interviewers will not like you to bring this to their company. When the inputs are in the hundreds of thousands this program will run very slow!

I will always provide a brute force solution if one exists & you should always solve the problem with one if you cannot come up with the optimized. A solution is better than no solution! 

BUT let's make sure that moving forward we also get some practice typing out the optimized solution because optimized will get you the job!

Remember to type it twice right away!